README.md

```diff
@@ -299,7 +299,7 @@ module.exports = [
 | [await-async-utils](docs/rules/await-async-utils.md)                             | Enforce promises from async utils to be awaited properly                                     | ![badge-angular][] ![badge-dom][] ![badge-marko][] ![badge-react][] ![badge-vue][] |                                                                     |     |
 | [consistent-data-testid](docs/rules/consistent-data-testid.md)                   | Ensures consistent usage of `data-testid`                                                    |                                                                                    |                                                                     |     |
 | [no-await-sync-events](docs/rules/no-await-sync-events.md)                       | Disallow unnecessary `await` for sync events                                                 | ![badge-angular][] ![badge-dom][] ![badge-react][]                                 |                                                                     |     |
-| [no-await-sync-queries](docs/rules/no-await-sync-queries.md)                     | Disallow unnecessary `await` for sync queries                                                | ![badge-angular][] ![badge-dom][] ![badge-marko][] ![badge-react][] ![badge-vue][] |                                                                     |     |
+| [no-await-sync-queries](docs/rules/no-await-sync-queries.md)                     | Disallow unnecessary `await` for sync queries                                                | ![badge-angular][] ![badge-dom][] ![badge-marko][] ![badge-react][] ![badge-vue][] |                                                                     | 🔧  |
```

Comment:
I think there are some conflicts here, can you resolve that @neriyarde?

Reply:
Yes, thanks.

Im getting some type errors in these files when im trying to solve the conflicts:
- lib/create-testing-library-rule/index.ts
- lib/rules/await-async-utils.ts
- tools/generate-configs/utils.ts
- lib/utils/types.ts
- lib/utils/compat.ts


I don't remember doing any changes in these files but i still get type errors.

Since I don't know these changes and i don't want to break anythig, can I get some help solving these from someone who's familiar with these files and type?